portfolio/src/main/java/com/google/sps/comment/Comment.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.comment;
+
+public final class Comment {
+    private final String comment;
+    private final long id;
+    private final long time;
+
+    public Comment(long id, String comment, long time) {
+        this.id = id;
+        this.comment = comment;
+        this.time = time;
```

Comment:
Probably better to call this `timestamp`, rather than `time`, which might be confused for an instance of some class in `java.time`.

Reply:
Sure. I will change it in the next commit.